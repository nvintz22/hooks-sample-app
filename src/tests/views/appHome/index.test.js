import React from 'react'
import { mount } from 'enzyme'
import enzymeSerializer from 'enzyme-to-json/serializer';

import AppHome from '../../../views/appHome/index';

expect.addSnapshotSerializer(enzymeSerializer);

describe('Home', () => {
    it('Should render home page', () => {
        const wrapper = mount(<AppHome />)
        expect(wrapper).toMatchSnapshot()
    }) 
})