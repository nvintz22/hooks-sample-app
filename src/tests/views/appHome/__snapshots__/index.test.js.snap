// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Should render home page 1`] = `
<Home>
  <h3>
    React@16.8 (Hooks)
  </h3>
  <small>
    Danniel Irvin
  </small>
  <br />
  <br />
  <form>
    <Styled(div)
      className="group"
    >
      <div
        className="group css-1c687hf"
      >
        <Styled(input)
          onChange={[Function]}
          required={true}
          type="text"
          value="Danniel Irvin"
        >
          <input
            className="css-zs2jp2"
            onChange={[Function]}
            required={true}
            type="text"
            value="Danniel Irvin"
          />
        </Styled(input)>
        <span
          className="highlight"
        />
        <Styled(div)
          className="bar"
        >
          <div
            className="bar css-utljbf"
          />
        </Styled(div)>
        <Styled(label)>
          <label
            className="css-1odazk5"
          >
            First name
          </label>
        </Styled(label)>
      </div>
    </Styled(div)>
  </form>
</Home>
`;
